modules/shared/src/main/scala/org/tessellation/security/Hash.scala

```diff
@@ -16,7 +18,9 @@ object hash {
 
   @derive(encoder, decoder, ordering, order, show)
   @newtype
-  case class Hash(value: String)
+  case class Hash(value: String) {
+    def getBytes: Array[Byte] = value.getBytes(StandardCharsets.UTF_8)
+  }
```

Comment:
I'd be careful with such changes, as we have a live production environment where we have to keep hash continuity.

Reply:
Also, such change imo should be a separate change (separate PR) fixing single or all the places where it's used. Otherwise, it would be really hard to pinpoint a single change that's breaking something (in general, not exactly this case).

Reply:
Yes, that's why I changed the two places that we're doing exactly that but left alone the two cases where it wasn't. I can revert if you still feel uncomfortable with it. At some point the question will be asked,
"Which of the two ways we're getting bytes is correct?"

Reply:
ok, got it :+1: